JetNews/app/src/main/java/com/example/jetnews/ui/interests/InterestsScreen.kt

```diff
@@ -510,6 +507,46 @@ private fun InterestsTabRow(
     }
 }
 
+/**
+ * Return state for interest sections grouped given the number of columns that fill the screen.
+ * Sections are distributed to balance the number of items on each column.
+ */
+@Composable
+private fun sectionsGroupedInColumnsAsState(
+    sections: List<InterestSection>,
+    columns: Int,
+    defaultDispatcher: CoroutineDispatcher = Dispatchers.Default
+): State<Array<MutableList<InterestSection>>> = produceState(
```

Comment:
We should definitely avoid putting a `Mutable***` object into a `State` (even though we won't modify it elsewhere).

If that list were to change, Compose wouldn't be able to track it.

Reply:
Done thanks!